site/content/docs/projects/custom-project.md

```diff
@@ -0,0 +1,16 @@
+###Customizing deployment projects
```

Comment:
We need to highlight the importance of saving the deployment project in version control because they are required for redeployments. If a user deploys with a deployment project the deployment project is no longer temporary.

Reply:
done